### YamlMime:UniversalReference
ms.openlocfilehash: 40b5d97358bd9cbe072188fac214f9276a6866e1
ms.sourcegitcommit: efa2d98deffe8a0d41a8d63f9f07aa720862e6ab
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 12/13/2018
ms.locfileid: "52038203"
items:
- uid: azure-arm-datalake-store.AdlsRuntimeException
  name: AdlsRuntimeException
  fullName: AdlsRuntimeException
  children: []
  langs:
  - typeScript
  type: interface
  summary: <span data-ttu-id="b263d-101">WebHDFS исключение, возникающее при произошла непредвиденная ошибка во время операции.</span><span class="sxs-lookup"><span data-stu-id="b263d-101">A WebHDFS exception thrown when an unexpected error occurs during an operation.</span></span> <span data-ttu-id="b263d-102">Возникает при возвращении ответа код ошибки 500 (Внутренняя ошибка сервера).</span><span class="sxs-lookup"><span data-stu-id="b263d-102">Thrown when a 500 error response code is returned (Internal server error).</span></span>
  extends:
    name: azure-arm-datalake-store.AdlsRemoteException
  package: azure-arm-datalake-store
references:
- uid: azure-arm-datalake-store.AdlsRemoteException
  name: AdlsRemoteException
  spec.typeScript:
  - name: AdlsRemoteException
    fullName: AdlsRemoteException
    uid: azure-arm-datalake-store.AdlsRemoteException
